build/pkgs/cython/dependencies

```diff
@@ -1,4 +1,4 @@
-$(PYTHON) | $(PYTHON_TOOLCHAIN)
+$(PYTHON) | $(PYTHON_TOOLCHAIN) pythran
```

Comment:
This introduces a circular dependency: cython -> pythran -> numpy -> cython

I tried building from scratch and failed at numpy with `ModuleNotFoundError: No module named 'Cython'` (before that, make decided to drop the dependency numpy <- cython so it attempted to build numpy before cython).

Reply:
Not sure if we still need the dependency pythran -> numpy

Reply:
> Not sure if we still need the dependency pythran -> numpy

It's only a runtime dependency. Not necessary to build (indeed void package builds without any dependency except setuptools)

Reply:
For the record: `beniget`, `gast` and `ply` are also not build dependencies of pythran.

I assume you want cython to depend on pythran for testing purposes ?